### PR TITLE
Add CI tests to drone

### PR DESCRIPTION
**Pull request by dpakach**
Add `drone.yml` file and tests to drone
related to #242

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    pull: true
+    environment:
+      - BROWSER=chrome
+      - SELENIUM_HOST=selenium
+      - SELENIUM_PORT=4444
+      - TEST_SERVER_URL=http://owncloud
+      - PLATFORM=Linux
+      - BEHAT_SUITE=${BEHAT_SUITE}
+    commands:
+      - make test-acceptance-webui
```
[the remark this answers is not in the document]
Comment (phil-davis) [reply .drone.yml]: Run.sh has new smarts in it to guess where be hat.yml might be found.
I forget exactly, but I think it "just works" (tm)

**Comment by individual-it** [comment]
@ownclouders rebase

**Comment by phil-davis** [comment]
https://travis-ci.org/owncloud/files_texteditor/jobs/454907915
```
$ sh -c "if [ '$TC' = 'other' ]; then bash apps/$APP/tests/travis/start_php_unit_tests.sh; fi"
Running PHP unit tests
Could not read "phpunit.xml".
The command "sh -c "if [ '$TC' = 'other' ]; then bash apps/$APP/tests/travis/start_php_unit_tests.sh; fi"" exited with 1.
```
Moving stuff around broke the Travis script.
The easy way to fix that here is to remove the PHPunit jobs from the Travis YML! (since we are going to delete the whole of the Travis YML in the next PR anyway)
@dpakach

**Comment by phil-davis** [comment]
For drone, it will need the new QA tarball tonight. That will have the ``FilesPageCRUD`` methods that were added to core today.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    when:
+      matrix:
+        TEST_SUITE: web-acceptance
+
+  codecov:
+    image: plugins/codecov:2
+    secrets: [codecov_token]
+    pull: true
+    paths:
+      - tests/output/clover.xml
```
Comment (individual-it) [inline .drone.yml]: I think we can use the file path in `files:`

**Pull request merged**